content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -448,11 +473,11 @@ Have feedback or questions? We welcome your thoughts at email_automation@sendgri
   </tr>
   <tr>
     <td>Ability to delete contacts from All Contacts or specific lists.</td>
```

Comment:
"Ability to delete contacts from your account or remove them from specific lists."

@negarzipan